ee/tabby-webserver/docs/api_spec.md

```diff
@@ -16,43 +16,58 @@ Get file or directory content from local repositories
 
 ```shell
 curl --request GET \
-  --url http://localhost:8080/repositories/https_github.com_TabbyML_tabby.git/resolve/
+  --url http://localhost:8080/repositories/git/lNrAwW/resolve/ \
+  -H "Authorization: Bearer {Token}"
```

Comment:
no need to ad authorization for all requests, document this at very beginning of the doc.